packages/expo-camera/src/CameraView.tsx

```diff
@@ -116,6 +116,21 @@ export default class CameraView extends Component<CameraProps> {
     return (await this._cameraRef.current?.getAvailablePictureSizes()) ?? [];
   }
 
+  /**
+   * Resumes the camera preview.
+   */
+
+  async resumePreview(): Promise<void> {
+    return this._cameraRef.current?.resumePreview();
+  }
+
+  /**
+   * Pauses the camera preview. It is not recommended to use `takePictureAsync` when preview is paused.
```

Comment:
What happens when you take the picture when it's paused? Maybe we should note this in `takePictureAsync` as well.

Reply:
Done 👍